pkg/graph/schema.graphql

```diff
@@ -1303,6 +1303,15 @@ input PlanOpsEvalAndLearningChanges @goModel(model: "map[string]interface{}") {
 
     status: TaskStatusInput
 }
+
+"""
+NDAAccepted represents whether a user has accepted an NDA or not. If accepted previously, there will be a datestamp visible
+"""
+type NDAAccepted {
```

Comment:
Pedantic, I know, but I think we should try and follow nouns for our types over verbs/adjectives. Something like `NDAAcceptance` might read a little more naturally? Or even `NDAAcceptanceInfo`, or maybe even just `NDAInfo`?

Reply:
Actually, I just scrolled down a bit and I think you already solved this! The table calls it `nda_agreement`, so maybe we can just call it `NDAAgreement` here?

Reply:
Probably worth a quick naming audit of `accept` vs. `agree` too, while we're talking about naming!

Reply:
Thanks @ClayBenson94! How do you feel about NDAInfo? I was avoiding calling it NDAAgreement, because this isn't the GO model with the same name (which was the original approach). 

And good call about agree v accept. It looks like in the vernacular of model forms, most places use agreement information etc. It probably makes sense to change accepted to agreed etc.

Reply:
Ah, that's true, since we're not returning the raw type that maps to the DB! Yeah, I'm good with `NDAInfo`!

Reply:
@ClayBenson94, thanks! Changes made and the test cases are updated